wsuwp-extended-events-calendar.php

```diff
@@ -354,15 +354,15 @@ public function community_events_submission_details( $errors ) {
 		$custom_message = ( array_key_exists( 'review-message', $options ) && '' !== $options['review-message']  ) ? $options['review-message'] : false;
 		$event_details = ( array_key_exists( 'review-details', $options ) && true === $options['review-details'] ) ? true : false;
 
-		if ( ! $custom_message && ! $event_details ) {
+		if ( ! $custom_message && ! $event_details && ! wp_verify_nonce( $_POST['_wpnonce'], 'ecp_event_submission' ) ) {
 			return $errors;
 		}
 
 		if ( is_array( $errors ) && 'update' === $errors[0]['type'] ) {
 
 			// Retrieve the ID of the submitted event.
-			preg_match( '!\d+!', $errors[0]['message'], $match );
-			$event_id = $match[0];
+			$event = get_page_by_title( $_POST['post_title'], OBJECT, 'tribe_events' );
```

Comment:
`$_POST['post_title']` gets passed directly to a SQL query via `$wpdb->prepare()` after this. That query is escaped, but still gives me some creeps. Can this be sanitized before the lookup with `sanitize_text_field()` or something?